### DISCUSSION TITLE
I get a call to undefined method after update, how to fix it?

### DISCUSSION THREAD

**Discussion by gdippe** [datenstrom/community#284] [Ask a question] [answered]
I'm stuck here after finally having to upgrade Yellow manually. If I leave the webinterface* files in the plugins folder editing etc works but the links look like [webinterfaceEdit] etc.
I get a blank page when I replace the webinterface* files with the new edit* files. In the Apache log file I get the below message.
Clearly I'm missing something but I cannot figure what! Help would be much appreciated!
PHP Fatal error:  Call to undefined method YellowToolbox::getMetaData() in /usr/local/www/data-dist/webbassocierade/luma/system/plugins/edit.php on line 932

**Comment by markseuffert** [comment, marked as answer]
The error message means the latest `system/plugins/core.php` is missing.

Download the latest version or run a [normal update](https://github.com/annaesvensson/yellow-update), then it should go away.

**Comment by gdippe** [comment]
I tried to update the usual way but that failed for some reason. Thats why I manually replaced the files with the updated ones. Right now I tried once again to replace core.php but I still get the same error. Is there any other file that is involved in this which might tigger the error?

**Comment by markseuffert** [comment]
Sorry, I don't know if there's another file. I would do the following: Double check that latest version of the core is in the file system, restart your web server to get rid of any PHP code that might be cached, install a brand new Yellow in a subfolder if nothing else helps.

**Comment by gdippe** [comment]
Thank you! I'll do that.
